### PR TITLE
feat: add processType property to app-info

### PR DESCRIPTION
**Pull request by apaleslimghost**

### CODE REVIEW

**Comment by apaleslimghost** [comment]
@rowanmanning i've added lambda support

File: packages/app-info/README.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-  * [License](#license)
-
+- [Usage](#usage)
+  - [`appInfo.commitHash`](#appinfocommithash)
+  - [`appInfo.environment`](#appinfoenvironment)
+  - [`appInfo.region`](#appinforegion)
+  - [`appInfo.releaseDate`](#appinforeleasedate)
+  - [`appInfo.releaseVersion`](#appinforeleaseversion)
+  - [`appInfo.systemCode`](#appinfosystemcode)
+  - [`appInfo.processType`](#appinfoprocesstypee)
```
Comment (rowanmanning) [inline packages/app-info/README.md:12]: **issue:** link is incorrect

```suggestion
  - [`appInfo.processType`](#appinfoprocesstype)
```

File: packages/app-info/README.md [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-    * [`appInfo.commitHash`](#appinfocommithash)
-    * [`appInfo.environment`](#appinfoenvironment)
-    * [`appInfo.region`](#appinforegion)
-    * [`appInfo.releaseDate`](#appinforeleasedate)
-    * [`appInfo.releaseVersion`](#appinforeleaseversion)
-    * [`appInfo.systemCode`](#appinfosystemcode)
-  * [Contributing](#contributing)
-  * [License](#license)
-
+- [Usage](#usage)
```
Comment (rowanmanning) [inline packages/app-info/README.md:5]: **suggestion:** there are a few Markdown styling changes in here which make the README diverge from the rest of the packages (e.g. spacing and changing `*` to `-`). I think there's some work for us to do to enforce the consistency we currently have but in the meantime could you revert these changes?

**Review by rowanmanning — COMMENTED** [review]
Sorry I somehow missed these small Markdown issues when I looked before. Code is all excellent obvs, great tests 👍

File: packages/app-info/README.md (hunk 2 above)
Comment (apaleslimghost) [reply packages/app-info/README.md:5]: gah, sorry, VS Code must have formatted it without me noticing

**Review by rowanmanning — APPROVED** [review]
Looks great 🎉 thanks!
